HilbertCurve.FCMacro

```diff
@@ -0,0 +1,246 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you please move to `ObjectCreation/HilbertCurve.FCMacro`?

Reply:
moved